lineapy/plugins/jinja_templates/dvc_dag_StagePerArtifact.jinja

```diff
@@ -0,0 +1,20 @@
+stages:
+{% for stage in STAGES %}
+    {{ stage["name"] }}:
+        cmd: python task_{{ stage["name"] }}.py
+    {%- if stage["deps"]|length > 0 %}
```

Comment:
Is this if statement misplaced? I think its one level too high and should only include lines 9-11

Reason i think that is that deps here is `deps=value.loaded_input_variables`.

This means if we don't have loaded_input_variables, `{{ MODULE_NAME }}.py`  and `task_{{ stage["name"] }}.py` aren't included as deps as I assume they should be.

Reply:
Yes, you are completely right... It was late when I realized that I forgot to include the python file dependencies...